### ISSUE TITLE
Page Transition - Feedbacks

### ISSUE THREAD

**Issue by rodolphebertozzo** [elementor/elementor#16113]
When we click on launch the animation, it is launched and we can preview it, but the behavior is repeated and remains blocked on the transition, and by pressing again on the button (to try to unblock the situation), the animation is reloaded but the page remains blocked. Also, from the moment the animation is defined and you reload the Transition Page part in the site settings, the animation remains blocked. Observe, it is more telling:
By the way, nice feature

https://user-images.githubusercontent.com/56270626/130508574-e0b727c1-555a-41fc-9974-a12287502976.mp4

**Comment by shilo-ey** [comment]
We are still working on this feature and will review your suggestions. feel free to open a new issue if needed
